File: 1833-maximum-ice-cream-bars/1833-maximum-ice-cream-bars.py
class Solution:
    def maxIceCream(self, costs: List[int], coins: int) -> int:
        sorted_cost, ans = sorted(costs), 0
        for cost in sorted_cost:
            if coins >= cost:
                coins -= cost
                ans += 1
            else:
                break
        return ans
        

File: 1833-maximum-ice-cream-bars/1833-maximum-ice-cream-bars.ts
function maxIceCream(costs: number[], coins: number): number {
    costs.sort((a,b) => a -b)
    let ans = 0;
    for(let cost of costs){
        if(coins >= cost){
            coins -= cost;
            ans++
        } else {
            break
        }
    }
    return ans
};